slatedb/src/cached_object_store/storage_fs.rs

```diff
@@ -122,7 +122,6 @@ impl FsCacheEntry {
             .await
             .map_err(wrap_io_err)?;
         file.write_all(&buf).await.map_err(wrap_io_err)?;
-        file.sync_all().await.map_err(wrap_io_err)?;
```

Comment:
I think we need to have checksums of the parts before doing this, which is a little tricky because you'd then need to read the whole part to validate the checksum

Reply:
I don't think there's a guarantee that the dirty pages are written in file order right?